### PR TITLE
#132 images are downloadble

### LINKED ISSUE #132: block: Download Images
https://www.volvotrucks.us/news-and-stories/press-releases/2023/february/vt-introduces-connected-vehicle-analytics-tool-to-maximize-vehicle-efficiency-and-productivity/

block name: Download Images

### PR DESCRIPTION
**Pull request by jonatan-lledo-netcentric**
Is needed to check the pages that have this block since the text "Here" must be on the same line as the rest of the sentence, otherwise, it will generate a button (check my demo page as example)

Fix #132 

Test URLs:
- Before: https://main--vg-volvotrucks-us--hlxsites.hlx.page/drafts/jlledo/press-releases/press-release-demo
- After: https://132-block-download-images--vg-volvotrucks-us--hlxsites.hlx.page/drafts/jlledo/press-releases/press-release-demo

### CODE REVIEW

**Review by Lakshmishri — CHANGES_REQUESTED** [review]
In mobile designs the images are too small, i.e screen size > 576 , existing site looks like this .
Screenshot 2023-03-27 at 09 52 13

**Comment by synox** [comment]
@jonatan-lledo-netcentric I seems to download the optimized images for the download. I think the JS should trip away any parameters from the image url to get the original.

**Review by synox — CHANGES_REQUESTED** [review]
@jonatan-lledo-netcentric I seems to download the optimized images for the download. I think the JS should trip away any parameters from the image url to get the original.

**Comment by jonatan-lledo-netcentric** [comment]
> 

I already saw that the link is broken because it missed the volvotrucks domain, so to have a working link, I prefer to not go page by page fixing the content. What can we do to get the original image then?

**Review by Lakshmishri — APPROVED** [review]

File: blocks/download-images/download-images.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+function addClickToLink(link) {
+  link.onclick = (e) => {
+    e.preventDefault();
+    // force to download
```
Comment (synox) [inline blocks/download-images/download-images.js:4]: Why is a new link created here? Can't we just use the original link?

**Comment by jonatan-lledo-netcentric** [comment]
> Also when clicking the download icon itself nothing happens.

this also fixed in the last update

**Comment by synox** [comment]
@jonatan-lledo-netcentric when it locally http://localhost:3000/drafts/jlledo/press-releases/press-release-demo wit works correctly. However on https://132-block-download-images--vg-volvotrucks-us--hlxsites.hlx.page/drafts/jlledo/press-releases/press-release-demo it just opens the image in the same tab. 

https://user-images.githubusercontent.com/2250964/229594130-196bb5fe-a48d-4ff5-bdc3-0e712a332abd.mov

**Comment by synox** [comment]
@jonatan-lledo-netcentric now the preview is downloaded instead of the original.

**Comment by jonatan-lledo-netcentric** [comment]
@synox I don't understand. I replicate the current behaviour. 

https://user-images.githubusercontent.com/117984204/230010523-61e70686-0e5d-40ca-a72d-34d4769d2215.mov

**Review by synox — APPROVED** [review]
looks good now

**Pull request merged**